packages/product-json-to-csv/src/map-headers.js

```diff
@@ -42,6 +42,11 @@ export default function mapHeaders(headers: Array<string>): Array<HeaderObj> {
         label: 'images',
         value: 'variant.images',
       }
+    if (header === 'attributes' || header === 'variant.attributes')
```

Comment:
```js
['attributes', 'variant.attributes'].includes(header)
```

1. easy to read
2. easy to extend
3. easy to extract for testing purposes
4. did I mention that it is easy to read?

Reply:
Just following the code structure, but I can make another PR for the refactoring.

https://github.com/commercetools/nodejs/blob/a6eca3d2619a6087e11b8301a42f4a9d3aeb5644/packages/product-json-to-csv/src/map-headers.js#L35-L49